guides/ageur-treasury.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Using agEUR as a treasury asset for your DAO or company
+---
+
+# agEUR as a treasury asset
+
+As we described [here](/guides/ageur-power-usecase.md), agEUR can be used by companies and DAOs to pay their contributors. But there is more!
+
+Angle’s agEUR can also be used as a treasury asset to diversify holdings and earn yield on top of EUR. 
+
+Most DAOs or companies with crypto treasuries hold a mix of BTC, ETH, and USD stablecoins. With agEUR, they have a way to diversify those reserves into one of the biggest fiat currency, through a decentralized stablecoin that has deep on-chain liquidity and is integrated with the best DeFi protocols.
+
+Holding agEUR in your treasury allows you to:
+
+- Diversify holdings
+- Earn yield on EUR
+- Keep enough capital to pay your european contributors, while enjoying yield on top of it.
+
+{% hint style="info" %}
+agEUR is available on Gnosis Safe, so you can hold some conveniently through your multi-sig.
```

Comment:
agEUR is a supported asset on Gnosis Safe and the Angle App integrates with Gnosis.